reledpar.dtx

```diff
@@ -804,16 +804,16 @@
 %   \end{Rightside}
 % \end{verbatim}
 % \subsection{Critical notes called on leftside but printed on right side}
-% Sometime, you need to print the critical notes on the right side, despite the refers to lemma of the leftside.
+% Sometimes, you need to print the critical notes on the right side, despite the references to the lemmas on the left side.
```

Comment:
Sometimes, you need to print the critical notes on the right side, despite *they refer* to the lemmas on the left sid